packages/opencensus-core/src/exporters/console-exporter.ts

```diff
@@ -55,19 +55,24 @@ export class ConsoleExporter implements Exporter {
    * @param span Ended span.
    */
   onEndSpan(span: modelTypes.Span) {
+    // Add spans of a trace together when root is ended, skip non root spans.
+    // publish function will extract child spans from root.
+    if (span.constructor.name !== 'RootSpan') {
```

Comment:
should we have an `isRoot` on the span btw?

Reply:
Done, 07b564e316be421a5c88e88d85759da735e33c12. PTAL